lib/ec2utils/VERSION

```diff
@@ -0,0 +1 @@
+4.9.95
```

Comment:
I think the version should move with the entire merge to the lowest common major version bump.

I.e. once things are all merged whatever the highest version between all separate utilities say 6.9.5 then this package becomes 7.0.0. Since this should be a major release. Thoughts?

Reply:
Yes, that's what will happen. This merged ec2deprecateimg which was at version 4.0.1 thus 4.9.95 to leave me room for other merges and a final cleanup at the end. Final cleanup will move things to ec2imgutils at which point all the imports will have to change.

Handling the version this way avoids me having to look at all the utils now and then figuring out the highest version of each individual util (not difficult of course) and then messing it up (which would be likely to happen). So what's checked in here is the highest for the merged sources according to the state of the merge process.